wordpress-mobile/WordPress-iOS
# Plans on JP: Analytics

Fixes #20692
pc8eDl-Vv-p2

- [ ] checkoutWebViewViewed not yet tracked, maybe could be done with the task that handles checkout?

### To test:

1. Log into free .com site and enable `Free to Paid plans dashboard card` feature flag 
2. `free_to_paid_plan_dashboard_card_shown` tracked when a dashboard card appears
3. `free_to_paid_plan_dashboard_card_tapped` tracked when a dashboard card tapped
4. `free_to_paid_plan_dashboard_card_menu_tapped` tracked when a dashboard card context menu tapped
5. `free_to_paid_plan_dashboard_card_menu_hidden` tracked when a dashboard card hidden
6. `domains_dashboard_select_domain_tapped` when domain selection is confirmed 
7.  `plan_selection_webview_viewed` when plan selection is presented

## Regression Notes
1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)

None

8. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:
